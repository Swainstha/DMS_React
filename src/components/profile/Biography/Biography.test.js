import testSnapFunction from '../../../util/testSnapFunction';
import Biography from './index'
import React from 'react';
const dummy_data= {
   data:{
       title:"title",
       icon:"icon.png", 
       desc:"this is a description", 
       userList:[{id:1},{id:2}]
   }
}

testSnapFunction("<Biography />", "Snapshot test for Biography ",<Biography {...dummy_data}/> )