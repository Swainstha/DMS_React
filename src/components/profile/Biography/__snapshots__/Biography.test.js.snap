// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Biography /> Snapshot test for Biography  1`] = `ShallowWrapper {}`;
